src/lib/VTXSPI/devVTXSPI.cpp

```diff
@@ -149,24 +152,40 @@ static void RfAmpVrefOff()
     RfAmpVrefState = 0;
 }
 
+static void setPWM()
+{
+#if defined(PLATFORM_ESP32)
+    if (GPIO_PIN_RF_AMP_PWM == 25 || GPIO_PIN_RF_AMP_PWM == 26)
+    {
+        dacWrite(GPIO_PIN_RF_AMP_PWM, vtxSPIPWM >> 4);
+    }
+    else
+    {
+        pwm.write(GPIO_PIN_RF_AMP_PWM, vtxSPIPWM);
+    }
+#else
+    analogWrite(GPIO_PIN_RF_AMP_PWM, vtxSPIPWM);
```

Comment:
We could even remove the defines for the 8285 since the spi vtx targets are now only esp32.

Reply:
Yes, there are a lot of defines that can be cleaned up, but I think I'll leave that for a later PR 😄